docs-ref-conceptual/get-started-tutorial-2-environment-syntax.md

```diff
@@ -0,0 +1,626 @@
+|az {"key":"value"} --debug | Missing single quotes | Unexpected token ':"value"' in expression or statement.
+|az {"key": "value"} --debug | Missing single quotes and contains an extra space | Error: Unexpected token ':' in expression or statement.
+
+# [Cmd](#tab/cmd)
+
+This example is **correct**.
+
+```azurecli-interactive
+az "{\"key\":\"value\"}" --debug
+```
+
+See what the Azure CLI is interpreting in the `Command arguments` line of the output.
+
+```output
+Command arguments: ['{"key":"value"}', '--debug']
+```
+
+These examples are all **incorrect**. Use the Cmd's `cls` command to remove console output between tests.
+
+|Incorrect format|Problem |Console output|
+|-|-|-|
+|az "{"key":"value"}" --debug | Missing escape characters | Command arguments: ['{key:value}', '--debug']
+|az '{"key":"value"}' --debug |Missing escape characters and is using single quotes where double quotes are expected.| Command arguments: ["'{key:value}'", '--debug']
+|az {"key":"value"} --debug | Missing escape characters and double quotes | Command arguments: ['{key:value}', '--debug']
+
+---
+
+### Use `echo` command
+
+Although `--debug` tells you exactly what the Azure CLI is interpreting, a second option is to return the value of an expression to your console. This method is very helpful when verifying the results of `--query` that is covered in detail in [Populate variables for use in scripts](./get-started-tutorial-3-use-variables.md).
+
+# [Bash](#tab/Bash)
+
+```azurecli-interactive
+strExpression='{"key":"value"}'
+echo $strExpression
+```
+
+```output
+{"key":"value"}
+```
+
+# [PowerShell](#tab/powershell)
+
+```azurecli-interactive
+$strExpression='{"key":"value"}'
+echo $strExpression
+```
+
+```output
+{"key":"value"}
+```
+
+# [Cmd](#tab/cmd)
+
+In Cmd, the `echo` command returns the literal string including escape characters. Compare the `--debug` and `echo` output of Cmd:
+
+|CLI|Output|
+|-|-|
+|az "{\"key\":\"value\"}" --debug | Command arguments: ['{"key":"value"}', '--debug']
+|set strExpression='"{\"key\": \"value\"}"' |
+| echo %strExpression% | "{\\"key\\": \\"value\\"}"
+
+---
+
+## Troubleshooting
+
+There are common errors when an Azure CLI reference command syntax is not written properly.
+
+* "Bad request ...{something} is invalid" might be caused by a space, single or double quotation mark, or lack of a quote.
+
+* "Unexpected token..." is seen when there is an extra space or quote.
+
+* "Invalid jmespath_type value" error often comes from incorrect quoting in the `--query` parameter.
+
+* "Unrecognized arguments" is often caused by an incorrect line continuation character.
+
+* "Missing expression after unary operator" is seen when a line continuation character is missing.
+
+* "Variable reference is not valid" is received when a string is not formatted properly often due to concatenation or a missing escape character.
+
+## Get more detail
```

Comment:
Consider changing to "Get more details" (make details plural throughout this section)